packages/keycloak-user-management/src/components/forms/UserForm/index.tsx

```diff
@@ -103,6 +103,10 @@ const UserForm: React.FC<UserFormProps> = (props: UserFormProps) => {
       lg: { offset: 6, span: 14 },
     },
   };
+  const status = [
```

Comment:
can we convert this into a Enums?

Reply:
We are passing status as `options` to `Radio.Group`